pkg/transforms/db/interfaces/store.go

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
```

Comment:
move these out of transforms package to internal/store/db... . transforms is for pipeline functions.